starfish/imagestack/imagestack.py

```diff
@@ -645,35 +636,44 @@ def apply(
             Function to apply. must expect a first argument which is a 2d or 3d numpy array and
             return an array of the same shape.
         group_by : Set[Axes]
-            Axes to split the data along.  For instance, splitting a 2D array (axes: X, Y; size:
+            Axes to split the data along.
+            `ex. splitting a 2D array (axes: X, Y; size:
             3, 4) by X results in 3 arrays of size 4.  (default {Axes.ROUND, Axes.CH,
-            Axes.ZPLANE})
+            Axes.ZPLANE})`
         in_place : bool
-            (Default False) If True, function is executed in place. If n_proc is not 1, the tile or
+            If True, function is executed in place. If n_proc is not 1, the tile or
             volume will be copied once during execution. If false, a new ImageStack object will be
-            produced.
+            produced. (Default False)
         verbose : bool
             If True, report on the percentage completed (default = False) during processing
         n_processes : Optional[int]
             The number of processes to use for apply. If None, uses the output of os.cpu_count()
             (default = None).
         kwargs : dict
             Additional arguments to pass to func
-        clip_method : Union[str, Clip]
-            (Default Clip.CLIP) Controls the way that data are scaled to retain skimage dtype
-            requirements that float data fall in [0, 1].
-            Clip.CLIP: data above 1 are set to 1, and below 0 are set to 0
-            Clip.SCALE_BY_IMAGE: data above 1 are scaled by the maximum value, with the maximum
-            value calculated over the entire ImageStack
-            Clip.SCALE_BY_CHUNK: data above 1 are scaled by the maximum value, with the maximum
-            value calculated over each slice, where slice shapes are determined by the group_by
-            parameters
+        clip_method : Union[str, :py:class:`~starfish.types.Clip`]
+
+            - Clip.CLIP (default): Controls the way that data are scaled to retain skimage dtype
```

Comment:
I think something a bit weird is happening with this list (Clip.CLIP is listed twice).